js/build/shortcode-ui.js

```diff
@@ -612,6 +618,13 @@ var editAttributeField = Backbone.View.extend( {
 			id: 'shortcode-ui-' + this.model.get( 'attr' ) + '-' + this.model.cid,
 		}, this.model.toJSON() );
 
+		// Handle legacy custom meta.
+		// Can be removed in 0.4.
+		if ( data.placeholder ) {
+			data.meta.placeholder = data.placeholder;
+			delete data.placeholder;
+		}
```

Comment:
I've no idea why this code has been added while compiling. I've removed in the new commit and it appears again.